syntax/scnvim.vim

```diff
@@ -0,0 +1,79 @@
+" Syntax highlighting for the scnvim post window
+"
+" By Mads Kjeldgaard
+" 2020-05-11
+"
+"
+" Building on the work of Alex Norman, David Granström and Stephen Lumenta for
+" SuperCollider, SCVIM and SCNVIM.
+
+scriptencoding utf-8
+
+" Check if syntax highlighting for the post window is active
+if !get(g:, 'scnvim_postwin_syntax_hl', 1)
+  finish
+end
+
+" Check if this syntax file has been loaded before
+if exists('b:current_syntax')
+	finish
+endif
+let b:current_syntax = 'scnvim'
+
+if (g:scnvim_postwin_syntax_hl == 1)
```

Comment:
We can remove this if-statement completely since we have the "early return" at the top of the script. Users will get an error if `g:scnvim_postwin_syntax_hl` is undefined here.

Reply:
This is a strange one. If i don't define that global variable at all, the syntax colouring is always triggered if I remove this if statement.

Reply:
Should say I am on Nvim 0.5 so there could be some differences in use there?

Reply:
The first if statement (with the early return) also supplies the default value with the `get(g: ...)`

I think that post window syntax hl should be the default, but in case anyone want's to disable it they can put `let g:scnvim_postwin_syntax_hl = 0` in their init.vim.

Oh, and before I forget! Since this is a new feature that variable should also be added to `doc/SCNvim.txt` with the default value of 1.

Reply:
Okay I think I have addressed all of this now, including the documentation. Thanks!